bundles/dip/Citrix/com.sapho.services.concur.ConcurService/0.7.7/i18n/ja.json

```diff
@@ -1,173 +1,173 @@
-    "f70872db-3c1f-4e66-a929-f0e68684fcb2" : "{{appr_login_id_report.last_name}} {{appr_login_id_report.first_name}}",
-    "dde0da2b-297f-4bd5-9252-6d909bd86777" : "総額",
-    "ae4797ac-3815-4dfa-9041-3ff021e8a4e1" : "Expense Details",
-    "9e142393-5888-4bb5-b456-fb249d268a5b" : "レポートの確認",
-    "ceddd6ca-18a0-477f-bd4d-7ef289f3d142" : "支出カテゴリ",
-    "5dd37eb9-772a-46cf-aec9-f556fd9813a6" : "自分の経費",
-    "6d0b9208-4963-4841-bb51-26d1aa9d302f" : "Report Status",
-    "24f2ed54-c287-4e38-9a18-85d613c77076" : "Type",
-    "5bcaf853-8e8d-4c0b-b8f8-f25fcc0a16b3" : "経費の検索",
-    "d57e6fdd-c655-4caa-a6b8-cb62f001da7f" : "Report ID",
-    "638dcad7-7f29-46fc-9258-5195dc6c3a15" : "{{format approved_amount \"decimal\"}} {{transaction_currency_code}}",
-    "a7814483-ee11-4391-afb8-df9393c618cd" : "Transaction Amount",
-    "f4490947-4501-4d48-a3c6-9b9b2185d515" : "状態",
-    "6c46135a-3e61-4a9d-980a-abe96b526477" : "Cash Advanced",
-    "e408abc3-b922-4ee1-b073-5f07927b9494" : "日付",
-    "6a3aae28-856e-4260-9ff5-9977c6d9fda1" : "経費が見つかりませんでした。",
-    "6dd5904d-632b-4dbe-8e34-6204371a9aeb" : "保留中のレポートを承認するか、差し戻します。 ",
-    "e1dfed7f-d2be-4265-8a3a-ff81aa2d296a" : "提出日",
-    "36ed4ba9-b76c-4b00-9acd-f4e7c9f0c06b" : "領収書",
-    "7eed4e88-40a2-448b-b889-caaa7d43c1c1" : "金額",
-    "7d18b289-bfc8-49eb-9b54-2c2ac8b17784" : "Expenses",
-    "fedc17e1-fa84-44e4-8aec-c19fee060d73" : "ベンダーの説明",
-    "480ee02e-15a8-4b74-ae12-b7093633a8c6" : "提出者",
-    "d05f8932-7523-4522-b5a1-5bd3aa7c30b2" : "Expense Report Details"
+  "a816e66c-8e1c-4430-a15b-4b59592d7256": {
+    "2e6177a8-7eca-426e-a07f-559d1d227de0": "経費の詳細",
+    "04ac0bbc-8215-4550-aaa0-f01b20232fb5": "ベンダー",
+    "067268a4-44f9-4d01-b0f6-7b8e0d3d2d86": "タイプ",
+    "d2d0d50b-1ef0-4c7a-aa2e-5b98493f473f": "{{format transaction_amount \"decimal\"}} {{transaction_currency_code}}",
+    "964d9d02-6de2-44b0-9bcf-0707e08f41c2": "{{format transaction_amount \"decimal\"}} {{transaction_currency_code}}",
+    "a465f98f-69b3-4435-b52d-444b5f6882ca": "日付",
+    "d39710e6-3410-47c7-a648-37123dc9631c": "日付",
+    "20b25a23-e490-4115-b4c5-d397ac3ba0d6": "ベンダー",
+    "2b8cf1f8-dbb6-474f-ba01-ed224558bd5a": "レポート番号",
+    "3a49ac0b-3532-4293-b113-477ce742531b": "取引金額",
+    "05f29eae-149e-4ee8-90cc-4fbf64d7cc75": "経費ポリシー",
+    "4facd8ae-fe98-439d-b704-a75a8cb01165": "経費",
+    "8ac505a2-84a9-4de5-ae3c-5d3a5d6c9f37": "{{format total \"decimal\"}} {{currency_code}}",
+    "dfdd1224-36e3-477e-8de7-91dcd3586357": "取引金額",
+    "c8a202f6-2903-48fc-ba88-3cdeec91ba5a": "表示するデータがありません",
+    "a6e03788-c4b7-4019-b714-700fce92e61d": "自分の経費",
+    "b5bb6768-e85a-4a3a-ac36-cb08e1f6e260": "{{format transaction_amount \"decimal\"}} {{transaction_currency_code}}",
+    "f70872db-3c1f-4e66-a929-f0e68684fcb2": "{{appr_login_id_report.last_name}} {{appr_login_id_report.first_name}}",
+    "dde0da2b-297f-4bd5-9252-6d909bd86777": "総額",
+    "ae4797ac-3815-4dfa-9041-3ff021e8a4e1": "Expense Details",
```

Comment:
Why not translate this string?

Reply:
@yiminl-ctx  this PR is just to sync the files to fix the g11n scanner issue. translations will be followed.